judging/scripts/generate_predictions.sh

```diff
@@ -25,16 +26,26 @@ predictions_file="$repo_dir/tasks/tasks.csv"
 generate_predictions_wrapper="$repo_dir/judging/generate_predictions_local.py"
 prediction_module="$HOME/work/predict.py"
 
+# Get latest validation module from base library (sandboxes in general have out of date base libraries)
```

Comment:
Extra logic to grab the validator module from `covid-xprize`. If we had some mechanism for updating the sandboxes' copies of this repo we wouldn't need to do this, but we don't have an easy way to do that.